Casks/vagrant-2.0.3.rb

```diff
@@ -0,0 +1,22 @@
+cask 'vagrant' do
```

Comment:
In order to maintain the ability to lock versions, you will need to name this `vagrant-2.0.3`. This is what allows us to prevent automatically updating to a newer version before it is vetted.